stack/templates/deployment.yaml

```diff
@@ -71,9 +71,15 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-          {{- with .Values.volumeMounts }}
+          {{- if or .Values.persistence.enabled .Values.volumeMounts}}
           volumeMounts:
-            {{- toYaml . | nindent 12 }}
+            {{- if .Values.persistence.enabled }}
+            - name: data
+              mountPath: {{ .Values.persistence.mountPath }}
+            {{- end }}
+            {{- if .Values.volumeMounts }}
+            {{- toYaml .Values.volumeMounts | nindent 8 }}
```

Comment:
Should `nindent` be changing from 12 to 8 here?

Reply:
Yes, your correct! Just fixed it